README.md

```diff
@@ -4,152 +4,183 @@
 
 > Easily enable fast AJAX navigation on any website (using pushState() + XHR)
 
-Pjax is **a standalone JavaScript module** that uses
-AJAX (XmlHttpRequest) and
-[pushState()](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Manipulating_the_browser_history)
-to deliver a fast browsing experience.
+Pjax is **a standalone JavaScript module** that uses [AJAX](https://developer.mozilla.org/en-US/docs/Web/Guide/AJAX) (XmlHttpRequest) and [pushState()](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Manipulating_the_browser_history) to deliver a fast browsing experience.
 
-_It allows you to completely transform the user experience of standard websites
-(server-side generated or static ones) to make them feel like they are browsing an app,
-especially for users with low bandwidth connection._
+_It allows you to completely transform the user experience of standard websites (server-side generated or static ones) to make them feel like they are browsing an app, especially for users with a low bandwidth connection._
```

Comment:
Something like _'...to make users feel like they are browsing an app, especially for those with low bandwidth connections.'_ would read a bit better here, I think